x-pack/plugins/case/server/services/alerts/index.ts

```diff
@@ -74,14 +73,14 @@ export class AlertService {
     return result;
   }
 
-  public async getAlerts({ request, ids, index }: GetAlertsArgs): Promise<AlertsResponse> {
-    if (!this.isInitialized) {
-      throw new Error('AlertService not initialized');
-    }
-
+  public async getAlerts({
+    scopedClusterClient,
+    ids,
+    indices,
+  }: GetAlertsArgs): Promise<AlertsResponse> {
     // The above check makes sure that esClient is defined.
-    const result = await this.esClient!.asScoped(request).asCurrentUser.search<AlertsResponse>({
-      index,
+    const result = await scopedClusterClient.search<AlertsResponse>({
+      index: [...indices].filter((index) => index !== ''),
```

Comment:
https://github.com/elastic/kibana/pull/88726/files#r573685286 and maybe is better to create a util private function in the class or in the file.